<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingerImgDlg</class>
 <widget class="QDialog" name="SingerImgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加背景</string>
  </property>
  <widget class="CButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnReturn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CButton" name="btnBackward">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnBackward.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CButton" name="btnForward">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnForward.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
